events/nats_config.go

```diff
@@ -55,7 +55,11 @@ type NATSConfig struct {
 
 // Configured checks whether the provider has been configured.
 func (c NATSConfig) Configured() bool {
-	return c.URL != "" || c.QueueGroup != ""
+	if c.QueueGroup == "" {
+		c.logger.Warn("NATS QueueGroup is not set. Subscriptions will not be durable.")
```

Comment:
This warning should go in `NewNATSConnection` function. This method is to determine whether this config has a configuration defined.